scanpipe/forms.py

```diff
@@ -117,6 +124,10 @@ def __init__(self, *args, **kwargs):
         name_field.widget.attrs["autofocus"] = True
         name_field.help_text = "The unique name of your project."
 
+    def clean(self):
```

Comment:
No need to override the `clean` method here. You can simply use the `clean_<fieldname>()` Django's API.
https://docs.djangoproject.com/en/3.1/ref/forms/validation/#cleaning-a-specific-field-attribute

Reply:
> Thanks for the PR.
> Could you add a unit test along the code changes?

Yes, will do that shortly.

Reply:
Hi @Sagarsangwan, any progress on this change?